en/rules.md

```diff
@@ -157,7 +155,7 @@ If you want to participate in a score hunt, [Join our Discord](https://discord.g
 *make sure to read the requirements of __each__ scorehunt. they can change depending on any of them.*
 
 the following will **ALWAYS** be true for scorehunts.
```

Comment:
For each scorehunt, the following will **ALWAYS** be applied:

Reply:
yeah what now